### PR TITLE
Skip [sig-network] test that was failing for a while but unnoticed

### PR DESCRIPTION
**Pull request by DennisPeriquet**
Regarding: https://bugzilla.redhat.com/show_bug.cgi?id=2042956

For now we'll skip since it's been failing for a while but unnoticed (as it was mistakenly classified as a flake before [a separate testSuite name was added to distinguish them](https://github.com/openshift/origin/pull/26755/files)).

### CODE REVIEW

File: pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go [hunk 1]
```diff
@@ -547,6 +547,22 @@ func init() {
...
+		"periodic-ci-openshift-release-master-ci-4.10-e2e-aws-ovn-upgrade",
+		"periodic-ci-openshift-release-master-ci-4.10-e2e-azure-upgrade-single-node",
+		"periodic-ci-openshift-release-master-ci-4.10-upgrade-from-stable-4.9-e2e-azure-ovn-upgrade",
+		"periodic-ci-openshift-release-master-ci-4.10-upgrade-from-stable-4.9-e2e-azure-upgrade",
+		"periodic-ci-openshift-release-master-nightly-4.10-upgrade-from-stable-4.9-e2e-metal-ipi-upgrade",
+	} {
+		testsRequiringHistoryRewrite[testCoordinates{
+			jobName:       jobName,
+			testName:      "[sig-network] pods should successfully create sandboxes by other",
+			testSuiteName: "openshift-upgrade-test",
```
Comment (dgoodwin) [inline pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go:563]: openshift-tests-upgrade

File: pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go [hunk 2]
```diff
@@ -555,8 +571,6 @@ func testShouldAlwaysPass(jobName, testName, testSuiteName string) string {
 		testName:      testName,
 		testSuiteName: testSuiteName,
 	}
-	// TODO remove this, but for now we have it to match our current hardcoded constants
-	coordinates.testSuiteName = ""
```
Comment (dgoodwin) [inline pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go:559]: Nervous about this but it does seem correct. We don't normally define a testSuite before this, now we do and this value should come through. I'm unclear why we were setting empty string prior to this patch though, it should be that by default.

File: pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go [hunk 3]
```diff
@@ -547,6 +547,22 @@ func init() {
...
 	}
+
+	for _, jobName := range []string{
+		"periodic-ci-openshift-multiarch-master-nightly-4.10-upgrade-from-nightly-4.9-ocp-remote-libvirt-s390x",
+		"periodic-ci-openshift-release-master-ci-4.10-upgrade-from-stable-4.9-e2e-aws-ovn-upgrade",
+		"periodic-ci-openshift-release-master-ci-4.10-e2e-aws-ovn-upgrade",
+		"periodic-ci-openshift-release-master-ci-4.10-e2e-azure-upgrade-single-node",
+		"periodic-ci-openshift-release-master-ci-4.10-upgrade-from-stable-4.9-e2e-azure-ovn-upgrade",
+		"periodic-ci-openshift-release-master-ci-4.10-upgrade-from-stable-4.9-e2e-azure-upgrade",
+		"periodic-ci-openshift-release-master-nightly-4.10-upgrade-from-stable-4.9-e2e-metal-ipi-upgrade",
```
Comment (dgoodwin) [inline pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go:558]: Some of these are not jobs we aggregate on but we didn't exactly make that clear this morning, I didn't even realize. The multiarch one for sure can go. Probably the single-node, and the metal job, none of those should be aggregated, but may want to wait to see what David thinks.

**Review by dgoodwin — COMMENTED** [review]
Looks good but we'll need your input and /approve @deads2k

File: pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go (hunk 1 above)
Comment (deads2k) [inline pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go:563]: this value needs to be set in all the skips above too, right?

File: pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go (hunk 3 above)
Comment (deads2k) [reply pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go:558]: Bring up the results in the test comparison view.  You only need to force skip on ones below about 50% pass rate.  And be sure that blocker bugs are open and don't let them be closed on, "no longer failing in CI".

File: pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go (hunk 1 above)
Comment (dgoodwin) [reply pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go:563]: I wonder which value needs to be set though, openshift-tests or openshift-tests-upgrade. Possible some have moved.

File: pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go (hunk 3 above)
Comment (dgoodwin) [reply pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go:558]: Test comparison view: https://datastudio.google.com/reporting/519bfc5e-6996-46bf-9269-0e2ca6d95eb4/page/qKmaC

Filter on test name contains "sandboxes by other".

Here's the blocker for at least aws-own: https://bugzilla.redhat.com/show_bug.cgi?id=2038481

The only one below 50% in that chart is aws ovn as far as I can tell from this.

File: pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go (hunk 1 above)
Comment (deads2k) [reply pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go:563]: The "Test Comparison" page can tell you.

Comment (dgoodwin) [reply pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go:563]: It's snipping the end of the name, so I can't quite see it unless I'm missing something.

Comment (dgoodwin) [reply pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go:563]: Can use the export link in top right of the form to get a CSV that will show you the full suite name.

Comment (dgoodwin) [reply pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go:563]: Or just hover :)

File: pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go (hunk 3 above)
Comment (dgoodwin) [reply pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go:558]: I think it's just periodic-ci-openshift-release-master-ci-4.10-upgrade-from-stable-4.9-e2e-aws-ovn-upgrade

We failed a gcp ovn but it was only off by one pass, 8 when we required 9, so that's not the same issue at all.

File: pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go (hunk 1 above)
Comment (dgoodwin) [reply pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go:563]: @DennisPeriquet so to be clear, check each of the tests above, look for them in the Test Comparison chart in data studio, find the ones not passing well (thus were added to ignore), verify their suite name, and add to these maps so each has a suite name.

Comment (DennisPeriquet) [reply pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go:563]: oh man, can't believe I got that wrong.  Thx.

Comment (DennisPeriquet) [reply pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go:563]: @deads2k when I put the testSuiteName in there, do I need to skip both the version that has the `"openshift-tests"`, `"openshift-tests-upgrade"` and `null` (I'm assuming `null` renders to `""`)?

Comment (DennisPeriquet) [reply pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go:563]: actually, the suite may also be something other than `openshift-tests` and `openshift-tests-upgrade`.

Comment (dgoodwin) [reply pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go:563]: i think you need to lookup which one is failing in the Test Comparison chart.

Comment (DennisPeriquet) [reply pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go]: yup, the "34.63% Working" one has Suite=openshift-tests-upgrade
